src/models/brightcove.js

```diff
@@ -155,22 +211,112 @@ class Brightcove extends Video {
 				if (titleEl) {
 					this.containerEl.removeChild(titleEl);
 				}
-				this.removePlaceholder();
-				this.addVideo();
+				this.containerEl.removeChild(this.placeholderEl);
+
+				this.el.style.display = 'block';
 				this.el.play();
 				this.el.focus();
 			});
 		}
-	}
 
-	removePlaceholder() {
-		this.containerEl.classList.remove('o-video--placeholder');
-		this.containerEl.removeChild(this.placeholderEl);
+		// Adds video soon so ads can start loading
+		this.addVideo();
+		// Hide it so it doesn't flash until the placeholder image loads
+		this.el.style.display = 'none';
 	}
 
 	getProgress() {
 		return this.el.duration ? parseInt(100 * this.el.currentTime / this.el.duration, 10) : 0;
 	}
+
+	playAdEventHandler() {
+		// Sets the styling now so the ad occupies the space of the video
+		this.adContainerEl.classList.add('o-video__ad');
+		// Initialize the video. Must be done via a user action on mobile devices.
+		this.el.load();
+		this.adDisplayContainer.initialize();
+
+		try {
+			// Initialize the ads manager. Ad rules playlist will start at this time.
+			this.adsManager.init(640, 360, google.ima.ViewMode.NORMAL);
```

Comment:
it seems that this is what setting player to get bigger that the original video.
Searching https://developers.google.com/interactive-media-ads/docs/sdks/html5/v3/apis#ima.AdsManager for clues on how to make it fir container